src/components/table/table.tsx

```diff
@@ -167,22 +168,47 @@ class Table<IDataCoordinates = any> extends React.Component<ITableProps<IDataCoo
       this.initGroupsProps(groups);
     }
 
+    let newOpenedTrees = openedTrees;
+
     if (prevProps.rows !== rows) {
+      newOpenedTrees = this.removeRowsFromOpenedTrees(openedTrees, rows);
```

Comment:
it's more a question of synchronization between rows and openedTrees. another name ?